cosmos/x/evm/keeper/msg_server.go

```diff
@@ -41,21 +41,13 @@ func (k *Keeper) EthTransaction(
 	ctx context.Context, msg *types.EthTransactionRequest,
 ) (*types.EthTransactionResponse, error) {
 	// Process the transaction and return the result.
-	result, err := k.ProcessTransaction(ctx, msg.AsTransaction())
+	receipt, err := k.ProcessTransaction(ctx, msg.AsTransaction())
```

Comment:
``ProcessTransaction`` returns an ``ExecutionResult``, maybe rename variable to ``executionResult`` for clarity

Reply:
This PR changes it to return receipt